graphql_client_codegen/src/codegen/enums.rs

```diff
@@ -23,7 +23,9 @@ pub(super) fn generate_enum_definitions<'a, 'schema: 'a>(
     );
     let normalization = options.normalization();
 
-    all_used_types.enums(query.schema).map(move |(_id, r#enum)| {
+    all_used_types.enums(query.schema)
+        .filter(move |(_id, r#enum)| !options.extern_enums().contains(&r#enum.name))
+        .map(move |(_id, r#enum)| {
```

Comment:
I assume this _just works_ because the generated modules already `use super::*`?

Reply:
Yes, it very much relies on that. It's simple and it works. :) Unless you think `use super::*` might be removed in the future, I don't really see this as a problem. But happy to look into alternatives if you think there could be a cleaner solution.